plugins/plant.cpp

```diff
@@ -0,0 +1,627 @@
+// Grow and remove shrubs or trees.
+
+#include <iostream>
+#include <iomanip>
+#include <map>
+#include <algorithm>
+#include <vector>
+#include <string>
```

Comment:
since you're organizing, system headers should go last, after `df/` headers. also you can probably get away with not including most of these since they're already included by other headers